oct/pages/base/base.py

```diff
@@ -4,6 +4,7 @@
 from abc import ABC
 from typing import Any, Union, Type, List, Optional
 
+import requests
```

Comment:
Won't it be better to import only Request and Session from requests?

Reply:
Done, thanks!